test/service/test_manifest.py

```diff
@@ -431,10 +431,13 @@ def test_manifest_zarr(self):
                 'specimen_from_organism_organ': 'brain'
             }
         ]
-        for entry in expected:
-            url = furl(entry['file_drs_uri'])
-            entry['file_drs_uri'] = url.set(scheme='drs',
-                                            host=config.drs_domain).url
+
+        def _file_drs_uri(drs_uri):
+            drs_uri = furl(drs_uri)
+            self.assertTrue(drs_uri.origin.startswith('drs://drs'), drs_uri.origin)
```

Comment:
We can't assume anything about the host-name other than it being a valid host name. I think this would break in `prod`. Try running the test against `prod`.

```suggestion
            self.assertTrue(drs_uri.origin.startswith('drs://'), drs_uri.origin)
```

And for that you can look at `.scheme` or `.protocol` I forgot what it's called.

Reply:
Not sure if you were asking me to replace the usage of `startswith` with `scheme` but that is what I did. However, scheme does not include the delimiter, so couldn't implement your suggestion verbatim.

> Try running the test against `prod`.

You were right, it did fail `prod`, thanks for catching that 😅 .